_data/accessibility-tests/tooltip.yml

```diff
@@ -0,0 +1,86 @@
+      contrast between the background and text is at least 4:5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+  - summary: Appropriate color contrast between tooltip and page background.
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the tooltip component itself and any adjacent background content is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct the tool tip or other content.
+    summary_additional: When you zoom to 200%, the tooltip content remains visible and does not overlap with other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+  - summary: Tooltip remains visible and functional when screen is magnified.
+    summary_additional: When you zoom to 200%, you can still read content in the tooltip and surrounding area easily.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Active tooltip content can be closed with a mouse or keyboard.
+    summary_additional: When you open a tooltip with a mouse, you can use the "escape" key to close it.
+    test_status: exception
+    status_details: |
+      You currently cannot dismiss a tooltip without moving focus away from the trigger component.
+    github_issue_number: 5909
+    github_issue_repo: uswds
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.13
+  - summary: Tooltips can be accessed with only a keyboard.
+    summary_additional: When you tab to an element that has a tooltip, the tooltip activates.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: The tool tip does not trap the user or move the user around the page unwillingly.
+    summary_additional: When you open a tooltip, you can navigate in and out of the trigger component without difficulty.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.2
+  - summary: Focus indicator is clearly visible around trigger element.
+    summary_additional: When you tab into an element with a tooltip, a visible focus indicator appears around that element.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader announces tooltip content.
+    summary_additional: |
+      When you focus on the tooltip trigger,
+      the screen reader announces the tooltip content.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.1
```

Comment:
@amycole501 @alex-hull @sarah-sch @finekatie 
I updated the summary and test status for 1.3.1 according to [this conversation](https://github.com/uswds/uswds/issues/5891#issuecomment-2088981813). I also removed the test for 4.1.2. Please let me know if you want any changes here!